cmd/vic-machine/create/create.go

```diff
@@ -943,6 +943,16 @@ func (c *Create) Run(cliContext *cli.Context) (err error) {
 		log.SetLevel(log.DebugLevel)
 		trace.Logger.Level = log.DebugLevel
 	}
+
+	// urfave/cli will print out exit in error handling, so no more information in main method can be printed out.
+	defer func() {
```

Comment:
the recover is in urfave/cli error handling. And it will also call os.Exit to hard exit everything, that's why I cannot have this printed out in main method.
I do want to have this common, but as this kind of defer method, with variable error, how to do it?